source/includes/windows/quickstart.rst

```diff
@@ -10,7 +10,17 @@ Quickstart for Windows
 
 This procedure deploys a :ref:`Single-Node Single-Drive <minio-installation-comparison>` MinIO server onto |OS| for early development and evaluation of MinIO Object Storage and its S3-compatible API layer. 
 
-For instructions on deploying to production environments, see :ref:`deploy-minio-distributed`.
+.. note::
+
+   MinIO has limited support for multi-drive Windows deployments due to behaviors and limitations of the NTFS filesystem.
+
+   This documentation only covers Single-Node Single-Drive deployment topologies on Windows.
+   Use Linux hosts for deploying Single-Node Multi-Drive or Multi-Node Multi-Drive MinIO topologies.
```

Comment:
I suggest doing ref links on these for quick access to the Linux docs. Sphinx interlinking should Just Work™, right?

Reply:
So I have _not_ been able to get this to work when there are overlapping refs.

The sphinx docs indicate it *should* work. In theory the syntax is correct. In practice it doesn't fire.

My guess is something in my custom code isn't completely correct for intersphinx. So external links will have to do for now.